notebooks/1_table_oriented.ipynb

```diff
@@ -0,0 +1,620 @@
+{
```

Comment:
<p>Let's provide <code>index=</code> to the DataFrame call above, so that we have an <code>Index</code> object here and not a RangeIndex.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>